addons/website_event/controllers/main.py

```diff
@@ -392,7 +431,10 @@ def registration_confirm(self, event, **post):
         registrations_data = self._process_attendees_form(event, post)
         registration_tickets = Counter(registration['event_ticket_id'] for registration in registrations_data)
         event_tickets = request.env['event.event.ticket'].browse(list(registration_tickets.keys()))
-        if any(event_ticket.seats_limited and event_ticket.seats_available < registration_tickets.get(event_ticket.id) for event_ticket in event_tickets):
+        if not post.get('event_slot_id') and any(
```

Comment:
I mean... considering the method docstring, shouldn't we also check for slots by calling our new magic method?

Seems a bit "too easy" to completely ignore it for our new use case.